roles/openshift_node/defaults/main.yml

```diff
@@ -140,6 +140,9 @@ openshift_node_use_nuage: "{{ openshift_node_use_nuage_default }}"
 openshift_node_use_contiv_default: "{{ openshift_use_contiv | default(False) }}"
 openshift_node_use_contiv: "{{ openshift_node_use_contiv_default }}"
 
+openshift_node_use_nsx_default: "{{ openshift_use_nsx | default(False) }}"
```

Comment:
Not sure what this var control - a default value for `openshift_node_use_nsx`? I think its fine to set `openshift_node_use_nsx` to False here, any user value would override that